File: frontend/src/models/Pizza/Pizza.ts
export default interface Pizza {
  id: number;
  name: string;
  description: string;
  imageData: string;
  recipeLink: string;
  timeCreated: Date;
  rating: number;
}

export const defaultPizza: Pizza = {
  id: 0,
  name: "",
  description: "",
  imageData: "",
  recipeLink: "",
  timeCreated: new Date(2000, 0, 1),
  rating: 3,
};

File: backend/API/Database/Models/Pizza.cs
namespace API.Database.Models
{
    public class Pizza
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string? Description { get; set; }
        public string? ImageData { get; set; }
        public string? RecipeLink { get; set; }
        public DateTime TimeCreated { get; set; }
        public int Rating { get; set; }
    }
}
